cloud/src/rate-limiter/rate_limiter.cpp

```diff
@@ -41,36 +46,92 @@ void RateLimiter::init(google::protobuf::Service* service) {
                 int64_t max_qps_limit = std::stoll(v.substr(p + 1));
                 if (max_qps_limit > 0) {
                     rpc_name_to_max_qps_limit[rpc_name] = max_qps_limit;
-                    LOG(INFO) << "set rpc: " << rpc_name << " max_qps_limit: " << max_qps_limit;
                 }
             } catch (...) {
-                LOG(WARNING) << "failed to set max_qps_limit to rpc: " << rpc_name
+                LOG(WARNING) << "failed to parse max_qps_limit to rpc: " << rpc_name
                              << " config: " << v;
             }
         }
     }
+    return rpc_name_to_max_qps_limit;
+}
+
+template <typename Callable>
+void for_each_rpc_name(google::protobuf::Service* service, Callable cb) {
```

Comment:
`rpc_name`s are fixed after starting the service, consider storing the names in a container which can be a member of MetaServiceImpl

Reply:
There's no need to add a container, this function is only called once when doing init.